docs/helm/eks-deployment.md

```diff
@@ -8,13 +8,210 @@ manage, and scale containerized applications using Kubernetes on AWS. EKS runs
 the Kubernetes management infrastructure for you across multiple AWS
 availability zones to eliminate a single point of failure.
 
-The Enterprise configuration will deploy the following system:
+## Architectures
+
+### Enterprise diagrams
+
+The Enterprise configuration will deploy the following system (Alfresco
+Transform Service and Alfresco Search Enterprise are detailed in the next
+diagrams):
+
+```mermaid
+graph LR
+
+classDef alf fill:#0b0,color:#fff
+classDef aws fill:#fa0,color:#fff
+classDef k8s fill:#326ce5,stroke:#326ce5,stroke-width:2px,color:#fff
+classDef thrdP fill:#e098a6,color:#000
+
+Client("👥 Clients")
+
+subgraph Helm enterprise
+  direction LR
+  PersistentVolumeClaim_activemq-default-pvc(PersistentVolumeClaim: activemq-default-pvc):::k8s
+  PersistentVolumeClaim_data-acs-postgresql(PersistentVolumeClaim: data-acs-postgresql):::k8s
+  PersistentVolumeClaim_data-sync-postgresql(PersistentVolumeClaim: data-sync-postgresql):::k8s
+  PersistentVolumeClaim_repository-default-pvc(PersistentVolumeClaim: repository-default-pvc):::k8s
+
+  Deployment_activemq(Deployment: activemq):::thrdP
+  Deployment_alfresco-cc(Deployment: alfresco-cc):::alf
+  Deployment_alfresco-dw(Deployment: alfresco-dw):::alf
+  Deployment_alfresco-repository(Deployment: alfresco-repository):::alf
+  Deployment_alfresco-sync-service(Deployment: alfresco-sync-service):::alf
+  Deployment_share(Deployment: share):::alf
+
+  StatefulSet_elasticsearch-master(StatefulSet: elasticsearch-master):::thrdP
+  StatefulSet_postgresql-sync(StatefulSet: postgresql-sync):::thrdP
+  StatefulSet_postgresql-acs(StatefulSet: postgresql-acs):::thrdP
+
+  Ingress_alfresco-cc(Ingress: alfresco-cc):::k8s
+  Ingress_alfresco-dw(Ingress: alfresco-dw):::k8s
+  Ingress_alfresco-repository(Ingress: alfresco-repository):::k8s
+  Ingress_alfresco-sync-service(Ingress: alfresco-sync-service):::k8s
+  Ingress_share(Ingress: share):::k8s
+end
+
+  subgraph AWS
+    EFS:::aws
+    S3:::aws
+    EBS:::aws
```

Comment:
EBS representing  individual  volumes you should have one for each ebs backed pvc

Reply:
the idea was more about EBS as a service to back a PVC rather than a volume - wouldn't make it more confusing adding more nodes?

Reply:
I don't think so. Actually it couild even make it more clear as each pvc could have a "straight" line to its dedicated volume